content/en/containers/kubernetes/control_plane.md

```diff
@@ -459,9 +460,9 @@ oc patch csv <datadog-operator.VERSION> -n <datadog-operator-namespace> \
   -p='[{"op": "add", "path": "/spec/install/spec/deployments/0/spec/template/spec/containers/0/args/-", "value": "--introspectionEnabled=true"}]'
 ```
 
-Since this feature is enabled by default, you can deploy a minimal DatadogAgent spec. 
+Since this feature is enabled by default, you can deploy a minimal DatadogAgent spec.
 
-Enable `features.clusterChecks.useClusterChecksRunners` to schedule checks there; otherwise, control plane checks will run on the Node Agent. 
+Enable `features.clusterChecks.useClusterChecksRunners` to schedule checks there; otherwise, control plane checks will run on the Node Agent.
```

Comment:
```suggestion
Enable `features.clusterChecks.useClusterChecksRunners` to schedule checks there; otherwise, control plane checks run on the Node Agent.
```